frontend/src/utils/date.ts

```diff
@@ -0,0 +1,11 @@
+import { formatISO, isValid, parseISO } from 'date-fns';
+
+export const serializeDate = (date: Date = new Date()) => {
+  const asString = formatISO(date, { representation: 'date' });
+
+  if (isValid(parseISO(asString))) {
```

Comment:
can this ever be false?

Reply:
Didn't change the logic. I guess formatISO could return null if date is null/undefined or some type other than Date.